genRSS.py

```diff
@@ -170,60 +156,53 @@ def build_item(link, title, guid = None, description="", pub_date=None, indent =
                 would give this tag:
                    <aTag>aValue</aTag>
 
-    Returns
-    -------
-    A string representing an RSS 2.0 item.
+    Returns:
+        A string representing an RSS 2.0 item.
+
+    Examples:
+        >>> item = build_item("my/web/site/media/item1", title = "Title of item 1", guid = "item1",
+        ...                  description="This is item 1", pubDate="Mon, 22 Dec 2014 18:30:00 +0000",
```

Comment:
`pubDate` argument is now called `pub_date`.

Please run `python genRSS.py --run-tests` after you update the code to make sure the tests pass.